### ISSUE TITLE
Installation error on ubuntu 16.04

### ISSUE THREAD

**Issue by rafa-acioly** [KELiON/cerebro#375]
- [x] I am on the [latest](https://github.com/KELiON/cerebro/releases/latest) Cerebro.app version
- [x] I have searched the [issues](https://github.com/KELiON/cerebro/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Ubuntu 16.04
- **Cerebro.app version**: 0.3
- **Relevant information from devtools** _(See above how to open it)_: 

## Issue
Trying to install on ubuntu 16.04 and i got the error:
>/home/dev/.local/share/applications/appimagekit-cerebro.desktop: error: value "public.app-category.productivity;" for key "Categories" in group "Desktop Entry" contains an unregistered value "public.app-category.productivity"; values extending the format should start with "X-"
Error on file "/tmp/.mount_giJBWb/cerebro.desktop": Failed to validate the created desktop file

Installation steps:
```sh
chmod a+x cerebro-0.3.0-x86_64.AppImage
./cerebro-0.3.0-x86_64.AppImage
```
> Would you like to integrate /home/dev/Downloads/cerebro-0.3.0-x86_64.AppImage with your system?
=> YES

The App run but i got the error above.

**Comment by KELiON** [comment]
@rafa-acioly it is fixed in #371, but is not released yet

**Comment by rohmanhm** [comment]
Waiting for release. Got the same problem here Ubuntu GNOME
